sklearn/covariance/tests/test_graph_lasso.py

```diff
@@ -88,6 +88,24 @@ def test_graph_lasso_iris():
         assert_array_almost_equal(icov, icov_R)
 
 
+@ignore_warnings(category=DeprecationWarning)
+def test_graph_lasso_2D():
+    # Hard-coded solution from Python skggm package
+    # obtained by calling `quic(emp_cov, lam=.1, tol=1e-8)`
+    cov_skggm = np.array([[3.09550269, 1.186972],
+                         [1.186972, 0.57713289]])
+
+    icov_skggm = np.array([[1.52836773, -3.14334831],
+                          [-3.14334831,  8.19753385]])
+    X = datasets.load_iris().data[:, 2:]
+    emp_cov = empirical_covariance(X)
+    for method in ('cd', 'lars'):
+        cov, icov = graph_lasso(emp_cov, alpha=.1, return_costs=False,
+                                mode=method)
+        assert_array_almost_equal(cov, cov_skggm)
+        assert_array_almost_equal(icov, icov_skggm)
+
+
```

Comment:
Since `graph_lasso` is deprecated and just calls out to `graphical_lasso` do we even need to add the test twice? 

I'd just test `graphical_lasso` but let's check with @agramfort or @GaelVaroquaux

Reply:
Sure, happy to remove the new test from `test_graph_lasso.py` if this is preferred

Reply:
agreed. not needed